src/app/components/pages/MyProgress.tsx

```diff
@@ -37,8 +38,44 @@ const statistics = {
   tagColWidth: "col-lg-12",
 };
 
-interface MyProgressProps extends RouteComponentProps<{ userIdOfInterest: string }> {
+const QuestionParts = ({ progress }: { progress: MyProgressState | undefined }) => {
+  return (
```

Comment:
Small one, but you can remove the braces and return to make it a bit more succinct, since you're not defining any variables or logic here. It's a matter of preference, though, so feel free to ignore this comment.